nimble/host/src/ble_hs_hci_cmd.c

```diff
@@ -1834,3 +1834,153 @@ ble_hs_hci_cmd_build_le_read_remote_feat(uint16_t handle, uint8_t *dst,
 
     return 0;
 }
+/*CTE Connectionless*/
+#if MYNEWT_VAL(CONNECTIONLESS_CTE_TX)
+static void
+ble_hs_hci_cmd_body_le_set_connless_cte_enable(uint8_t adv_handle ,
+        uint8_t enable, uint8_t *dst)
+{
+    dst[0] = adv_handle;
+    dst[1] = enable;
+}
+
+void ble_hs_hci_cmd_build_le_set_connless_cte_enable(uint8_t adv_handle,
+        uint8_t enable, uint8_t *dst, int dst_len)
+{
+    BLE_HS_DBG_ASSERT(dst_len >= BLE_HCI_SET_CONNLES_CTE_ENABLE_LEN);
+
+    ble_hs_hci_cmd_body_le_set_connless_cte_enable(adv_handle, enable, dst);
+}
+
+static int ble_hs_hci_cmd_body_le_set_connless_cte_params(
+        const struct hci_connless_cte_params *cte, uint8_t *dst)
+{
+    BLE_HS_DBG_ASSERT(cte != NULL);
+    dst[0] = cte->adv_handle;
```

Comment:
could you add parameters checking here as we are doing for other commands?

Reply:
Hello @rymanluk 
do i need to add checks here or to remove this code from ble_hs_hci_cmd.c ?
as i see in Use packed structures for HCI commands and events #507
this code will be removed and checks will be added in ble_gap.c 
i already developed code in ble_gap.c for Constant Tone Extension 
so i can remove this code and once this pr is accepted i will open new pr for CTE in GAP